### PR TITLE
New infrastructure agent release 1.52.0 release notes

### PR DESCRIPTION
**Pull request by alvarocabanas**
Release notes for infrastructure agent 1.52.0: https://github.com/newrelic/infrastructure-agent/releases/tag/1.52.0

### CODE REVIEW

File: src/content/docs/release-notes/infrastructure-release-notes/infrastructure-agent-release-notes/new-relic-infrastructure-agent-1520.mdx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+## Changed
+* logging config: wrapped Record attribute key and value in double quotes [#1827](https://github.com/newrelic/infrastructure-agent/pull/1827)
+* logging config: Add log warning msg when logging too many files in [#1823](https://github.com/newrelic/infrastructure-agent/pull/1823)
+* enable FluentBit installation for arm64 AmazonLinux 2023 in [#1824](https://github.com/newrelic/infrastructure-agent/pull/1824)
+* add new attribute host.id in connect metadata in [#1838](https://github.com/newrelic/infrastructure-agent/pull/1838)
+* Update dependency newrelic/nri-flex to v1.11.0 in [#1839](https://github.com/newrelic/infrastructure-agent/pull/1839)
```
Comment (ally-sassman) [inline src/content/docs/release-notes/infrastructure-release-notes/infrastructure-agent-release-notes/new-relic-infrastructure-agent-1520.mdx:14]:
```suggestion
## Changed

* Added logging config: Wrapped Record attribute key and value in double quotes [#1827](https://github.com/newrelic/infrastructure-agent/pull/1827)
* Added logging config: Add log warning msg when logging too many files in [#1823](https://github.com/newrelic/infrastructure-agent/pull/1823)
* Enabled FluentBit installation for arm64 AmazonLinux 2023 in [#1824](https://github.com/newrelic/infrastructure-agent/pull/1824)
* Added new attribute host.id in connect metadata in [#1838](https://github.com/newrelic/infrastructure-agent/pull/1838)
* Updated dependency newrelic/nri-flex to v1.11.0 in [#1839](https://github.com/newrelic/infrastructure-agent/pull/1839)
```

**Review by ghost — DISMISSED** [review]

File: src/content/docs/release-notes/infrastructure-release-notes/infrastructure-agent-release-notes/new-relic-infrastructure-agent-1520.mdx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+subject: Infrastructure agent
+releaseDate: '2024-04-02'
+version: 1.52.0
+---
+
+A new version of the agent has been released. Follow standard procedures to [update the infrastructure agent](https://docs.newrelic.com/docs/infrastructure/install-configure-manage-infrastructure/update-or-uninstall/update-infrastructure-agent). We recommend you upgrade the agent regularly and at a minimum every 3 months. As of this release, the oldest supported version is [Infrastructure agent 1.39.2](https://docs.newrelic.com/docs/release-notes/infrastructure-release-notes/infrastructure-agent-release-notes/new-relic-infrastructure-agent-1392/).
+
+## Changed
+
+* Added logging config: Wrapped Record attribute key and value in double quotes [#1827](https://github.com/newrelic/infrastructure-agent/pull/1827)
```
Comment (ally-sassman) [inline src/content/docs/release-notes/infrastructure-release-notes/infrastructure-agent-release-notes/new-relic-infrastructure-agent-1520.mdx:11]:
```suggestion
* Added logging config: Wrapped Record attribute key and value in double quotes in [#1827](https://github.com/newrelic/infrastructure-agent/pull/1827)
```

**Review by ghost — APPROVED** [review]

**Comment by ally-sassman** [comment]
@alvarocabanas Can you please sign the CLA? Thanks!

**Comment by alvarocabanas** [comment]
> @alvarocabanas Can you please sign the CLA? Thanks!

Done.

**Pull request merged**